doc/cephadm/install.rst

```diff
@@ -23,6 +23,10 @@ Requirements
 Any modern Linux distribution should be sufficient.  Dependencies
 are installed automatically by the bootstrap process below.
 
+.. warning::
+
+  Only podman versions that are 2.0.0 and higher work with Ceph Pacific, with the exception of podman version 2.2.1, which does not work with Ceph Pacific. kubic stable is known to work with Ceph Pacific, but it must be run with a newer kernel.
```

Comment:
not sure if we have a table or not. but it'd better to show this as a matrix -- i.e. a table.

Reply:
We'll need all the podman z-stream release version numbers in order to create a table.

Reply:
Hmm, the thing is that the current/latest ceph works around the podman 2.2.1 problems, so pacific is okay.. it's older octopus version that struggle with 2.2.1... most importantly, they can't upgrade to a later octopus that works (sigh).  I suggest

- we leave off the mention of 2.2.1 here
- we include discussion of 2.2.1 problems (and how to install kubic build) in octopus upgrade notes

Reply:
Can you move this sentence to https://docs.ceph.com/en/latest/cephadm/compatibility/#compatibility-with-podman-versions and just link this section from here?

Reply:
> ping

I'm on it, it's just at the bottom of queue of things.

Reply:
I'm sorry. I'm so behind things.

Reply:
If anyone owes an apology, it's me. (But I don't either.) So let's forgive ourselves.

Anyway: I have made the change. Excelsior!